packages/cbioportal-frontend-commons/src/components/scatterPlot/ScatterPlot.tsx

```diff
@@ -0,0 +1,368 @@
+import autobind from 'autobind-decorator';
```

Comment:
this is new code or its moved over from existing?

Reply:
This is new, but mostly inspired by `shared/components/plots/ScatterPlot.tsx` and `pages/studyView/charts/scatterPlot/StudyViewDensityScatterPlot.tsx`. Ideally we should be able to unify all these scatter plots in a generic way, but it would take more time and effort so I decided not to try that in this PR.

Reply:
hm okay